frontend/src/pages/user/home/Lessons/hooks/useLessons.js

```diff
@@ -4,25 +4,33 @@ import { useDispatch, useSelector } from "react-redux";
 
 import { authenticate } from "../../../../../helpers/auth";
 import { getLessons } from "../../../../../helpers/api";
```

Comment:
- move out api from `helpers` directory. and create a new directory name `api`. 
- group APIs by module. please follow the example below:
- create a base api for **adminApi.js** and **userApi.js**
```
>>frontend
   >>src
      >>api
          base.js
          adminApi.js
          userApi.js
          lessonApi.js
          .. and so on
```
the **baseApi.js** should contain an _axios instance_. And the other APIs should use the **baseApi.js**. For example below:
```
import API from './userApi';
import UserAPI from './base';

const LectureApi = {
  fetchSomething: (payload) => {
    const options = {
      method: 'GET',
      url: `/api/sample?page=${payload.page}`,
      params: {
        ...payload,
      },
    };

    return API.request(options);
  }
}

export default LectureApi;
```